examples/http-certification/upgrade-to-update-call/README.md

```diff
@@ -102,21 +100,15 @@ actor Http {
 
 ## Testing the canister
 
-Start DFX:
+To test the canister, you can use [`dfx`]([/docs/current/developer-docs/getting-started/install](https://internetcomputer.org/docs/current/developer-docs/getting-started/install) to start a local instance of the replica and deploy the canister:
 
 ```shell
 dfx start --background --clean
-```
 
-Deploy the Rust canister:
-
-```shell
+## Deploy the Rust canister
 dfx deploy http_certification_upgrade_to_update_call_rust_backend
-```
-
-Or deploy the Motoko canister:
 
-```shell
+## Deploy the Motoko canister
```

Comment:
I'm not sure what the advantage of this change is, could you elaborate on why you'd prefer all of these commands to be in the same codebox with comments separating them?

Similarly as above though, I'd note that the copy to clipboard button is unusable in this case and I think separate codeboxes is more useful for developers.

Reply:
Reverted this change